Automattic/wp-calypso
# Store Data: Move currencies & locations to the data-layer

This PR does some refactoring to move the locations & currency state to use the data-layer code. It also moves the files into the `data` folder (which then needs to be updated everywhere, hence the 35 files changed 😱) – this matches the standard we use in calypso, that data-layer code should line up to the API structure, and it made sense to have the redux handlers in the same folder structure.

**To test:**

- Run the tests
- Check places where locations and currencies are used:
  - payment settings screen
  - update store address
  - create an order, edit the customer address

There should be no visual changes here.
